src/docfx/lib/msgraph/MicrosoftGraphAccessor.cs

```diff
@@ -90,5 +92,26 @@ private async Task<T> SendRequest<T>(string api, Func<Task<T>> func)
                 _syncRoot.Release();
             }
         }
+
+        private static X509Certificate2 GetCertificateFromStoreByThumbprint(string certThumbprint)
+        {
+            using var store = new X509Store(StoreLocation.LocalMachine);
+            try
+            {
+                store.Open(OpenFlags.ReadOnly);
+
+                var matchedCerts = store.Certificates.Find(X509FindType.FindByThumbprint, certThumbprint, false);
```

Comment:
We are using self-signed cert in this scenario, so `validOnly` should set to `false`